### ISSUE TITLE
mysql float import error

### ISSUE THREAD

**Issue by coseyo** [siddontang/go-mysql-elasticsearch#147]
When the mysql field type is "float" and the first row value is 1.00. After exec the 
`bin/go-mysql-elasticsearch -config "./etc/river.toml"`
the field type in es will be "long", this bug will just appear only in some cases such as the int value like "1.00", "2.00".

**Comment by siddontang** [comment]
@coseyo 

Do you define the field as double in your ES mapping explicitly?

**Comment by coseyo** [comment]
No, just let the go-mysql-elasticsearch do it automaticly. You mean that I should create the mapping first?

**Comment by siddontang** [comment]
I guess ES covert 1.0 to long 1 implicitly. Maybe you can create the mapping at first and see whether this problem occurs or not.

**Comment by coseyo** [comment]
Yeah. I think so.
Creating mapping first, It is ok now. Besides, is there a solution for solving this "bug".
Thanks for reply!

**Comment by siddontang** [comment]
@coseyo 

Here we really send a double value in the JSON format. Maybe we can let ES know it explicitly, but I don't know now.

/cc @EagleChen do you have any idea?

**Comment by xxxlo71** [comment]
@coseyo

where can i setting ES mapping explicitly

**Comment by coseyo** [comment]
> @coseyo
> 
> Where can i setting ES mapping explicitly

You can setting ES map before you use it. So you can do by the head plugin or ES api, and so on.
